### PR TITLE
Exercise 9 of chapter 11 completed.

### PR DESCRIPTION
**Pull request by Htocki**

### CODE REVIEW

File: bin/11_9/src/Main.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  return flag == "-tB" || flag == "--to-binary";
+}
+
+bool IsToTextual(const std::string_view& flag) {
+  return flag == "-tT" || flag == "--to-textual";
+}
+
+bool IsFlag(const std::string_view& flag) {
+  return IsToBinary(flag) || IsToTextual(flag);
+}
```
Comment (sialiuk) [inline bin/11_9/src/Main.cpp]: pass string_view only by value

File: bin/11_9/src/Main.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      }
+      std::vector<int> values;
+      std::copy(
+        std::istream_iterator<int> { textual },
+        std::istream_iterator<int> { },
+        std::back_inserter(values));
+
+      std::ofstream binary { "file.bin", std::ios::binary };
+      for (auto value : values) {
+        binary.write(AsBytes(value), sizeof(int));
```
Comment (sialiuk) [inline bin/11_9/src/Main.cpp]: Can be combined:
const auto begin = std::istream_iterator<int> { textual };
const auto end = std::istream_iterator<int> { };
std::for_each(begin, end, [](const auto value){ ... })

**Review by sialiuk — CHANGES_REQUESTED** [review]

File: bin/11_9/src/Main.cpp (hunk 1 above)
Comment (Htocki) [reply bin/11_9/src/Main.cpp]: Fixed.

File: bin/11_9/src/Main.cpp (hunk 2 above)
Comment (Htocki) [reply bin/11_9/src/Main.cpp]: Will be implemented.

**Review by sialiuk — APPROVED** [review]

**Pull request merged**